datamap/bloom/src/main/java/org/apache/carbondata/datamap/bloom/BloomCoarseGrainDataMap.java

```diff
@@ -436,4 +436,9 @@ public String toString() {
   public void finish() {
 
   }
+
+  @Override public int getNumberOfEntries() {
```

Comment:
Move this method to available abstract class .

Reply:
done